docs/developer-docs/integrations/sns/integrating/frontend-integration.md

```diff
@@ -0,0 +1,18 @@
+# SNS frontend integration
+## Overview
+There are different reasons why you might want to interface with an SNS project.
+For example, you might want to integrate some of the SNS DAO functionality in the frontend of you dapp canister such that the dapp users can also vote on relevant SNS proposals (that goveran and change the dapp) directly in the same application.
```

Comment:
> in the frontend of you dapp canister

Typo (you vs your)
I would also shorten and just say "your dapp", drop "canister"

> goveran

Typo

Reply:
Thanks. fixed locally and will be included in next commit.